plugins/woocommerce/tests/e2e-pw/tests/customize-store/assembler/color-picker.spec.js

```diff
@@ -287,9 +287,9 @@ test.describe( 'Assembler -> Color Pickers', () => {
 	for ( const [ colorPaletteName, colors ] of Object.entries(
 		colorPalette
 	) ) {
-		test( `Color palette ${ colorPaletteName } should be applied`, async ( {
+		test.only( `Color palette ${ colorPaletteName } should be applied`, async ( {
```

Comment:
I think that we should remove `only` too:
```suggestion
		test( `Color palette ${ colorPaletteName } should be applied`, async ( {
```